.env.example

```diff
@@ -0,0 +1,2 @@
+#put this in .env.local
+NEXT_PUBLIC_BASE_API_URL='https://api.realdevsquad.com'
```

Comment:
What's `.env.example`?

Reply:
To give an idea to a new contributor to set up the repo env file, as I had ignored the .env.* files,

Reply:
`.env.local` takes higher priority than env specific variables and if someone wants to override the env variables, they should be playing with this file and this one should be in gitignore